elastic/indexer/insta_face/main/main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/elastic/go-elasticsearch/v7"
+	"github.com/elastic/go-elasticsearch/v7/esutil"
+
+	elasticsearch_inserter "github.com/codeuniversity/smag-mvp/elastic/indexer"
+	indexer "github.com/codeuniversity/smag-mvp/elastic/indexer/insta_face"
+	"github.com/codeuniversity/smag-mvp/kafka/changestream"
+	"github.com/codeuniversity/smag-mvp/models"
+	"github.com/codeuniversity/smag-mvp/service"
+	"github.com/codeuniversity/smag-mvp/utils"
+)
+
+const indexCreateBody = `
+{
+	"mappings" : {
+			"properties" : {
+				"encoding_vector": {
+					"type": "binary",
+					"doc_values": true
+				},
+				"post_id": {
+					"type": "integer"
+				},
+				"x": {
+					"type": "integer"
+				},
+				"y": {
+					"type": "integer"
+				},
+				"width": {
+					"type": "integer"
+				},
+				"height":{
+					"type": "integer"
+				}
+			}
+	}
+}
+`
+const esIndex = "faces"
```

Comment:
Let's move the index names to the elastic package - since we need them for the search